src/examples/algorithms/HelloSP_TP.cpp

```diff
@@ -1,96 +0,0 @@
-/* ---------------------------------------------------------------------
```

Comment:
Actually, imho the new file still handles both more elegantly, with DEBUG_LEVEL=0-1 it's your "hotgym", with debug >4 it's same as Hello_SP_TP output-wise. ....?